administration/console.md

```diff
@@ -15,15 +15,26 @@ The console offers the option to:
 
 ## Accessing the Console
 
-Accessing the console depends on the start mode of openHAB.
-The manually start using shell/batch script ends directly in the console.
+Accessing the console depends on how openHAB was started.
+When started manually using shell or batch script, openHAB naturally transitions directly to the console.
+When started as a service, the console can be reached via local SSH.  
```

Comment:
I'd not start to give a list with all possible OS's or something like that. The user is still encouraged to think for him/herself 😄  

Just one idea:

> When started in interactive mode using the provided command line scripts, openHAB naturally transitions directly to the console prompt.
> When started as a service (e.g. when installed from our package repository), openHAB is running as background process.
> In both cases the console can be reached via Secure Shell connection ([SSH](https://en.wikipedia.org/wiki/Secure_Shell)).

One other thing: I think most users nowadays are using some kind of openHAB service and will connect to the Console via SSH. Maybe we should move the first sentence further down anyhow?